lib/Console/Application.php

```diff
@@ -34,6 +34,8 @@ public function __construct(Configuration $configuration = null)
         $this->add(new RunCommand());
         $this->add(new ReportCommand());
 
+        gc_disable();
```

Comment:
Move to the run command? Does not make sense to disable this for the report generation command.